apcd-cms/src/apps/utils/static/utils/js/checkForBlankInputs.js

```diff
@@ -0,0 +1,6 @@
+const formTextInputs = document.querySelectorAll('input[type=text]');
+formTextInputs.forEach(i => i.addEventListener('change', () => {
```

Comment:
Need to add something here that also checks the text fields within the hidden html for entities and contacts when more than one is added. It's letting me submit with blank second inputs in the second contact and second entity.